contents/blog/eu-cloud-hosting.md

```diff
@@ -0,0 +1,89 @@
+---
+date: 2022-10-17
+title: How we built EU Cloud hosting, from idea to launch
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Inside PostHog", "Startups"]
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/posthog-eu-blog.png
+featuredImageType: full
+---
+
+We recently launched EU Cloud, providing all the benefits of cloud-hosted PostHog with data staying in the EU. We thought it would be interesting to go over why and how we built EU Cloud hosting, from idea to launch.
+
+## Why EU Cloud hosting?
+
+PostHog provides self-hosting options. Anyone can use PostHog and manage the infrastructure entirely how they want. This allows them to be more compliant with relevant laws in their industry or region. One of the big laws requiring compliance is [GDPR](/docs/integrate/gdpr) in Europe. Any time you see one of those annoying cookie banners, you’re seeing the results of GDPR. 
```

Comment:
```suggestion
PostHog provides self-hosting options. Anyone can use PostHog and manage the infrastructure entirely how they want. This enables them to be more compliant with relevant laws in their industry or region. One of the big laws requiring compliance is [GDPR](/docs/integrate/gdpr) in Europe. Any time you see one of those annoying cookie banners, you’re seeing the results of GDPR. 
```

For future reference: Andy disagrees, but I'm nearly always going to change `allows` to `enables` in order to make the tone more empowering and vigorous, less permissive.

Reply:
```suggestion
PostHog provides self-hosting options. Anyone can use PostHog and manage the infrastructure entirely how they want. This allows them to be more compliant with relevant laws in their industry or region. One of the big laws which teams across Europe need to comply with is [GDPR](/docs/integrate/gdpr) (General Data Protection Regulations). Any time you see one of those annoying cookie banners, you’re seeing the results of GDPR. 
```